src/qt/pivx/governancewidget.cpp

```diff
@@ -142,7 +142,24 @@ void GovernanceWidget::onVoteForPropClicked(const ProposalInfo& proposalInfo)
 
 void GovernanceWidget::onCreatePropClicked()
 {
-    if (!walletModel || !governanceModel) return;
+    if (!walletModel || !governanceModel || !clientModel) return;
+
+    if (!governanceModel->isTierTwoSync()) {
+        inform(tr("Please wait until the node is fully synced\""));
+        return;
+    }
+
+    // Do not allow proposals submission 1440 blocks away (1 day) from the next superblock
+    // The budget finalization could have been submitted and the user would never know it, losing the first superblock.
+    // And, do not allow submission until we are 120 blocks (2 hours) after the superblock
+    // just to be sure that the node passed the default chain reorg depth.
+    // future: customizable future superblock height selection (for now, we are automatically using the next superblock).
+    int height = clientModel->getLastBlockProcessedHeight();
+    int nextSuperblock = governanceModel->getNextSuperblockHeight();
+    if (nextSuperblock - 1440 < height && height < nextSuperblock + 120) {
```

Comment:
This does not work as intended I believe.
`height < nextSuperblock + 120` is always true because `nextSuperblock` is always greater than `height` (if it's passed, `getNextSuperblockHeight()` gives the *next* one).

Reply:
true. Done, have reworked it.